core/dispatcher/src/main/scala/whisk/core/container/ContainerPool.scala

```diff
@@ -310,8 +315,7 @@ class ContainerPool(
      * This call can be slow but not while locking data structure so it does not interfere with other activations.
      */
     def putBack(container: Container, delete: Boolean = false)(implicit transid: TransactionId): Unit = {
-        info(this, s"putBack returning container ${container.id}  delete = $delete")
-
+        info(this, s"putBack returning container ${container.id}  delete = $delete   slack = ${slack()}")
```

Comment:
yes - why not?  the more we can track all the state variables at every transition point the easier it will be to debug.